Ejercicios_Varios.py

```diff
@@ -108,3 +108,5 @@ def repetir(L,n):
 print(repetir([1,2,3],7))
 
 #Mas lineas
+
+#Esto lo incluyo el Dev1 
```

Comment:
No me gusta que no pusiste tilde
```suggestion
#Esto lo incluyoo el Dev1 
```